prow_config.yaml

```diff
@@ -28,59 +28,32 @@ workflows:
     params:
       installIstio: true
       workflowName: deployapp-istio
+  ### GCP IAP
   - py_func: kubeflow.kubeflow.ci.kfctl_e2e_workflow.create_workflow
     name: kfctl-go-iap-endpoint
     job_types:
+      - presubmit
```

Comment:
I don't think we want to run this particular test on presubmit.

The difference between this particular test and kfctl_go_test is that this instance is running test_endpoint which verifies that the IAP endpoint is accessible.

I don't think we want to run that on presubmit. Only postsubmit.

An alternative way to skip the test on presubmits would be using pytest annotations but that's not what we do right now.

Reply:
we are already running endpoint test in presubmit: https://github.com/kubeflow/kubeflow/blob/master/prow_config.yaml#L60